freqtrade/strategy/resolver.py

```diff
@@ -78,6 +82,14 @@ def _load_strategy(
             # Add extra strategy directory on top of search paths
             abs_paths.insert(0, extra_dir)
 
+        if validators.url(strategy_name):
+            temp = tempfile.mkdtemp("freq", "strategy")
+            abs_paths.insert(0, temp)
+            name = os.path.basename(urlparse(strategy_name).path)
+            urlretrieve(strategy_name, os.path.join(temp, name))
+            Path(os.path.join(temp, "__init__.py")).touch()
```

Comment:
If you import pathlib Path (the new stuff) - why use os.path to join?
[pathlib.Path.joinpath()](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.joinpath) should work too...

Reply:
still getting familiar with python. I will update it accordingly.